lookout/style/format/visualizer/server.py

```diff
@@ -122,11 +124,13 @@ def return_features() -> Response:
     labeled_indices = {id(vnode): i for i, vnode in enumerate(vnodes_y)}
     app.logger.info("returning features of shape %d, %d" % X.shape)
     app.logger.info("length of rules: %d", len(rules))
+    # TODO (zurk): Add refuse_to_predict to visualization
```

Comment:
I am not familiar with FrontEnd technologies, so I leave this todo for @m09 if you are so kind to help me with this. :) 
As soon as pure `y_pred` can have negative values I think this PR breaks visualization tool.